validators/terravaloper1vqnhgc6d0jyggtytzqrnsc40r4zez6tx99382w/profile.json

```diff
@@ -1,6 +1,10 @@
 {
-  "contact": {
-    "email": "",
-    "telegram": ""
+  "contact": { 
+      "email": "devops@kysenpool.io",
+      "telegram": "https://t.me/KysenPool",
+      "tweeter":  "https://twitter.com/kysenpool"
+  },
+  "notifications": {
+      "email": "devops@kysenpool.io" 
   }
 }
```

Comment:
remove extra bracket

Reply:
sorry, can't tell which bracket is extra

Reply:
It looks like Github complaining there is not a newline at the end of the file (not an extra bracket), add newline to the end of profile.json